data/fields/seamark/rescue_station/category.json

```
@@ -0,0 +1,19 @@
+{
+    "key": "seamark:rescue_station:category",
+    "type": "semiCombo",
+    "label": "Amenities",
```

Hm… _Amenities_ sounds a bit too unspecific here. Maybe the following would be better?
```suggestion
    "label": "Life Saving Equipment",
```